README.md

```diff
@@ -35,28 +41,75 @@ Before you can run the bot yourself, you'll need to set up an application on Dis
 
 ### Configuring
 
-High Quality Bot expects a file called `config.cfg` to be in the same directory as the .jar file, and won't run without it. The format is simply one `key=value` declaration per line. Currently available options to set are:
+High Quality Bot expects a file called `config.cfg` to be in the same directory as the .jar file, and won't run without
+it. The format is simply one `key=value` declaration per line. `exampleconfig.cfg` provides a sample configuration.
+Currently available options to set are:
 * `token` - Your Bot account's OAuth token
 * `prefix` - The desired character (or string of characters if you're a crazy person) used to invoke commands.
+* `ownerid` - The User attached to this ID is automatically given permission to use all commands.
+* `allowed` - This is a list of command names separated by commas that users are allowed to use by default.
+* `disallowed` - Users are by default explicitly disallowed to use these commands.
+* `allowedHasPrecedence` - If true, any commands in both `allowed` and `disallowed` are considered allowed.
```

Comment:
I think what I may do down the road is set things up so that if the priority isn't set, it's based around the Role's rank in Discord. That way by additionally ensuring that either `allowed` or `disallowed` is set to `*` in every group, Discord's behavior can be emulated.